packages/app-server-issuer/src/exchange-provider/config.ts

```diff
@@ -0,0 +1,6 @@
+interface Config {
+  cellAmount: number;
+  sudtArgs: string;
+  initCapacity: number;
+  sudtExchangeRate: number;
```

Comment:
using rate may encounter issues with the precision of floating point numbers, we can use a config like this


```sh
# 1 sUDT = 100 CKB
# the 0x... is args of the sUDT
# the decimals with the sUDT is 18
EXCHANGE_SUDT_AMOUNT_0x12345678=1000000000000000000 # 1e18
EXCHANGE_CKB_AMOUNT_0x12345678=10000000000 # 100e8
```

```ts
// 1 sUDT = 100 CKB
// assume the decimals of the sUDT is 18

const exchangeCkbAmount = BigNumber.from(process.env.EXCHANGE_CKB_AMOUNT_0x12345678)
const exchangeSudtAmount = BigNumber.from(process.env.EXCHANGE_SUDT_AMOUNT_0x12345678)

const exchangeableCkbAmount = BigNumber.from(readyForExchangeSudtAmount)
  .mul(exchangeCkbAmount)
  .div(exchangeSudtAmount)
```

and we can define the exchange price in a `.env` like this

Reply:
> using rate may encounter issues with the precision of floating point numbers, we can use a config like this
> 
> ```shell
> # 1 sUDT = 100 CKB
> # the 0x... is args of the sUDT
> # the decimals with the sUDT is 18
> EXCHANGE_SUDT_AMOUNT_0x12345678=1000000000000000000 # 1e18
> EXCHANGE_CKB_AMOUNT_0x12345678=10000000000 # 100e8
> ```
> 
> ```ts
> // 1 sUDT = 100 CKB
> // assume the decimals of the sUDT is 18
> 
> const exchangeCkbAmount = BigNumber.from(process.env.EXCHANGE_CKB_AMOUNT_0x12345678)
> const exchangeSudtAmount = BigNumber.from(process.env.EXCHANGE_SUDT_AMOUNT_0x12345678)
> 
> const exchangeableCkbAmount = BigNumber.from(readyForExchangeSudtAmount)
>   .mul(exchangeCkbAmount)
>   .div(exchangeSudtAmount)
> ```
> 
> and we can define the exchange price in a `.env` like this

good idea!